core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -651,6 +656,18 @@ private ExecutionModel parseCreateTable(GenericLexer lexer, SqlExecutionContext
         throw errUnexpected(lexer, tok);
     }
 
+    private void parseLikeTableName(GenericLexer lexer, CreateTableModel model) throws SqlException {
+        CharSequence tok;
+        tok = tok(lexer, "table name");
+        model.setLikeTableName(nextLiteral(GenericLexer.assertNoDotsAndSlashes(GenericLexer.unquote(tok), lexer.lastTokenPosition()), lexer.lastTokenPosition()));
+        tok = tok(lexer, ")");
+        if(!Chars.equals(tok, ")"))
+            throw errUnexpected(lexer, tok);
+        tok = optTok(lexer);
+        if(tok !=null && !Chars.equals(tok, ";"))
```

Comment:
nit: weird formatting:
```suggestion
        if (tok != null && !Chars.equals(tok, ";"))
```

Reply:
fixed